File: src/main/java/com/github/insanusmokrassar/iobject/exceptions/WriteException.java
package com.github.insanusmokrassar.iobject.exceptions;

import java.io.IOException;

public class WriteException extends IOException {

    public WriteException(String message) {
        super(message);
    }

    public WriteException(String message, Throwable cause) {
        super(message, cause);
    }
}

File: src/main/kotlin/com/github/insanusmokrassar/IObjectK/exceptions/WriteException.kt
package com.github.insanusmokrassar.IObjectK.exceptions

import java.io.IOException

/**
 * Exception which can be thrown in operations with writing in IOutputObject
 */
class WriteException : IOException {

    constructor(message: String) : super(message) {}

    constructor(message: String, cause: Throwable) : super(message, cause) {}
}
